doc-content/enterprise-only/release-notes/rn-760-fixed-issues-ref.adoc

```diff
@@ -0,0 +1,106 @@
+[id='rn-750-fixed-issues-ref']
+= Fixed issues in {PRODUCT} 7.6.0
+
+{PRODUCT} 7.6.0 provides increased stability and fixed issues listed in this section.
+
+ifdef::PAM[]
+[NOTE]
+====
+{PRODUCT_DM} fixed issues apply to {PRODUCT_PAM} as well. For a list of {PRODUCT_DM} 7.6.0 fixed issues, see the https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.6/html-single/release_notes_for_red_hat_decision_manager_7.6/index[_Release Notes for Red Hat Decision Manager 7.6_]
+====
+
+== Installer
+
+* {JWS}-only datasource configuration is missing elements [https://issues.redhat.com/browse/RHPAM-2428[RHPAM-2428]]
+
+== {CENTRAL}
+
+* Cloning from a remote git repostory that requires credentials does not work [https://issues.redhat.com/browse/RHPAM-2531[RHPAM-2531]]
+
+* After execution of a multi-instance subprocess, nodes are not blurred even though the process instance is completed [https://issues.redhat.com/browse/RHPAM-2472[RHPAM-2472]]
+
+* An error dialog appears when you import a sample project twice [https://issues.redhat.com/browse/RHPAM-2434[RHPAM-2434]]
+
+* You cannot upload or download files that contain spaces or ampersands (`&`) in name [https://issues.redhat.com/browse/RHPAM-2368[RHPAM-2368]]
+
+* Performance issues occur when you open assets with Project Explorer [https://issues.redhat.com/browse/RHPAM-2359[RHPAM-2359]]
+
+* Roles permissions do persisted and can reset after some time [https://issues.redhat.com/browse/RHPAM-2342[RHPAM-2342]]
+
+* JavaScript error when trying to modify action columns in guided decision table [https://issues.redhat.com/browse/RHPAM-2286[RHPAM-2286]]
```

Comment:
A JavaScript error occurs when you try to modify...
when trying - dangling modifier